### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by horlah**
This PR works with changes in this UI-Library PR: https://github.com/frain-dev/ui-library/pull/5

### CODE REVIEW

File: web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.component.html [hunk 1]
```diff
@@ -183,16 +188,17 @@ <h3>Configuration</h3>
 						</button>
 					</li>
 					<div class="color__grey font__14px font__weight-500 font--nowrap margin-left--auto padding-right__14px">
-						<span [hidden]="activeTab !== 'events'">
+						<span *ngIf="events" [hidden]="activeTab !== 'events'">
```
Comment (Oluwadaminiola) [inline web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.component.html:191]: why did you add *ngIf when there's [hidden]

Comment (horlah) [reply web/ui/dashboard-component/projects/convoy-dashboard/src/lib/convoy-dashboard.component.html:191]: I added it because I only want to show the pagination data if events data exists, else it'll throw errors. @Oluwadaminiola

File: web/ui/dashboard/angular.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
 						"tsConfig": "tsconfig.app.json",
 						"inlineStyleLanguage": "scss",
 						"assets": ["src/favicon.ico", "src/assets"],
-						"styles": ["src/styles.scss", "node_modules/convoy-ui/scss/main.scss"],
+						"styles": ["src/styles.scss", "/Users/emmanuelaina/projects/ui-library/scss/main.scss"],
```
Comment (Oluwadaminiola) [inline web/ui/dashboard/angular.json:33]: @horlah can you make a pr with the changes made on the UI library? I already setup a deploy preview on netlify so we can preview the changes on the UI library before publishing

Comment (horlah) [reply web/ui/dashboard/angular.json]: @Oluwadaminiola fixed now.

**Review by Oluwadaminiola — APPROVED** [review]
